extensions/sql-database-projects/src/models/tree/projectTreeItem.ts

```diff
@@ -67,55 +67,54 @@ export class ProjectRootTreeItem extends BaseProjectTreeItem {
 	 * Processes the list of files in a project file to constructs the tree
 	 */
 	private construct() {
+		// folders
+		for (const folder of this.project.folders) {
```

Comment:
is this needed? The folders are also getting added below on line 117

Reply:
Ah, I was moving the folder reading up here, and 117 must've been re-added during a merge.